modules/database/src/std/softIoc/softMain.cpp

```diff
@@ -228,7 +229,7 @@ int main(int argc, char *argv[])
                 std::cout<<"# End "<<argv[optind]<<"\n";
 
             epicsThreadSleep(0.2);
-            loadedDb = true;    /* Give it the benefit of the doubt... */
+            ranScript = true;    /* Assume the script has done any necessary initialization */
```

Comment:
Since we don't really need to know which of the two happened we could combine the two flags into one and rename it to something a bit more meaningful like `workloaded`.

Reply:
The `loadedDb` flag is being separately tested shortly after this line to decided whether to call `iocInit()`.